package.json

```diff
@@ -68,7 +62,6 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-mocha": "^5.3.0",
     "eslint-plugin-react": "^7.14.2",
-    "prepush": "^3.1.11",
```

Comment:
@samshull Random comment location here to allow threaded messages in github.

1. `prepush` has security vulnerability and was being replaced in another PR (linked in this PR description) with a precommit hook.
2. Why dictate a users's workflow or force them to use `--no-verify`.  The proper time to verify what someone has done is passing tests is at PR/merge time and no on every commit or push. If a developer wants to commit frequently small chunks of code, that should actually be encouraged. If a developer wants to use TDD and commit an initial set of breaking tests, that should be encouraged/allowed. If a developer wants to push frequently to avoid losing their work, they should be allowed to do so.

Reply:
I think doing the way proposed here (no prepush) is more of a "trust but verify" model, which is generally a good thing. You encourage the developer to run the test suite, and trust that they do when necessary. And then when a PR is created, you verify that it passes by running it again.

Reply:
There does not appear to be master branch protections on this repo. If we are to rely on PR testing, we should start by changing those rules first. And while I would prefer not to be a devil's advocate here, neither push, nor PR testing will prevent this package from being published to npm with bad code, so I would argue that we should be more concerned with prepublish actions than either of these.

Reply:
prepublish sounds like a great idea! Would you be willing to spin up a PR to propose that? :)

Reply:
As for branch protection rules, I just got those started with 2 required reviews; once the GH Actions PR gets merged, we can also set those up as required. 👍🏻

Reply:
(remember to use `prepublishOnly` to avoid the silly npm behavior of running `prepublish` after _installs_)